simplq/src/components/common/CreateJoinForm/CreateJoinForm.jsx

```diff
@@ -46,7 +46,7 @@ const CreateJoinForm = (props) => {
       <div className={styles['input-box']}>
         <InputField
           placeholder="Queue Name"
-          value={textFieldValue}
+          value={textFieldValue || ''}
```

Comment:
There was a warning popping up in the CreateJoinForm component on the browser console. This is to silence it according to [this](https://stackoverflow.com/questions/47012169/a-component-is-changing-an-uncontrolled-input-of-type-text-to-be-controlled-erro) answer.